options.html

```diff
@@ -39,6 +39,11 @@
 
 <div id="page">
     <div id="side_bar">
+        <div class="tab_row" data-block-sel="#maincontent_general">
+            <div class="category sidebar_entry">
+                <div data-locale-text="general">General</div>
```

Comment:
For some reason, here, or somewhere else where `general` is used as locale, gives an error that it cannot find `general` in `Localization`, but that's simply false...

Reply:
I should be `options.general` not `general`. Fixed now.